src/sentry/utils/suspect_resolutions/metric_correlation.py

```diff
@@ -0,0 +1,52 @@
+from datetime import timedelta
+from typing import List
+
+from sentry import tsdb
+from sentry.models import Group
+
+
+def is_issue_error_rate_correlated(
+    resolved_issue: Group, candidate_suspect_resolution: Group
+) -> bool:
+    if resolved_issue is None or candidate_suspect_resolution is None:
+        return False
+
+    resolution_time = resolved_issue.resolved_at
+
+    start_time = resolution_time - timedelta(hours=8)
+    end_time = resolution_time + timedelta(hours=8)
```

Comment:
This seems like we're casting a wide net. Can we be more precise with the time range?

https://docs.google.com/document/d/1BjykhZcMH5UCxlYh6mzbxA_LDyuVojotYvLxz5Q95Zs/edit?disco=AAAAdNyAf60